docs/getting-started.md

```diff
@@ -125,9 +125,16 @@ cd $Env:ProgramFiles\containerd\
 # - cni bin_dir and conf_dir locations
 Get-Content config.toml
 
+# Optional step to add the binaries (containerd.exe, ctr.exe) in $env:Path
+$Path = [Environment]::GetEnvironmentVariable("PATH", "Machine") + [IO.Path]::PathSeparator + $PWD.Path
```

Comment:
Is the current PWD path the right one?  should we use `$Env:ProgramFiles\containerd`?

Reply:
It's correct but it's relative to the previous command; I think better off to just make it absolute. Will fix.